shop/controllers/feedback.py

```diff
@@ -38,4 +46,19 @@ def update_feedback(cls, feedback_pk, product, title, content, images=None):
     def delete_feedback(cls, feedback_pk):
         FeedbackDAL.delete_feedback(cls.get_feedback(feedback_pk))
 
+        # TODO Sending email to feedback author
+
         return
+
+    @classmethod
+    def delete_feedback_images(cls, feedback_pk):
+        FeedbackDAL.delete_images(cls.get_feedback(feedback_pk))
+
+    @classmethod
+    def validate_images_pk_to_delete(cls, feedback, images_pk_to_delete):
+        if len(images_pk_to_delete) > FeedbackDAL.get_feedback_images_count(feedback):
+            raise serializers.ValidationError({'images_to_delete': 'Too many images!'})
```

Comment:
how can this be caused?

Reply:
If more numbers are transmitted than the number of feedback images

Reply:
what steps does the user need to perform to send to the backend such values?